backend/models/invoice.js

```diff
@@ -478,16 +475,8 @@
 
 	};
 
-	schema.methods.getPDF = function (options) {
-		options = options || {};
-
-		if (options.regenerate || !this.pdf) {
-
-			return this.generatePDF();
-
-		} else {
-			return Promise.resolve(this.pdf.buffer);
-		}
+	schema.methods.getPDF = function () {
```

Comment:
make routes/invoice call generatePDF instead? or just rename generatePDF to getPDF